src/app/shared/pipes/hours-from-minutes/hours-from-minutes.module.ts

```diff
@@ -0,0 +1,10 @@
+import { HoursFromMinutesPipe } from './hours-from-minutes.pipe';
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@NgModule({
+  declarations: [HoursFromMinutesPipe],
+  exports: [HoursFromMinutesPipe],
+  imports: [CommonModule],
+})
+export class HoursFromMinutesModule {}
```

Comment:
You went to extreme, probably one pipes module would be enough, usually pipes are simple :)